src/bigint.h

```diff
@@ -38,6 +38,12 @@ class Bigint
     Bigint operator*(long long const &);
     Bigint &operator*=(int const &);
 
+    //Division
+    bool is_even();
+    private:
+    std::vector<Bigint> divide(Bigint q);   // returns quotient(index[0]) and remainder(index[1]).
+    public:
+    std::vector<Bigint> operator/(Bigint q); // interface for divide() function.
```

Comment:
divide() can stay as is, but `/` operator should return just Bigint so syntax like this would work `2 * 5 / 3`.